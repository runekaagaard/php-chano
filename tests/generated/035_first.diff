001- t
001+ Fatal error: Call to undefined method DtlIter::first() in /home/rune/workspace/php-misc/php-dtl-iterator/tests/django-generated/035_first.php on line 4