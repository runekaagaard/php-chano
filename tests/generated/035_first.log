
---- EXPECTED OUTPUT
t
---- ACTUAL OUTPUT
Fatal error: Call to undefined method DtlIter::first() in /home/rune/workspace/php-misc/php-dtl-iterator/tests/django-generated/035_first.php on line 4
---- FAILED
