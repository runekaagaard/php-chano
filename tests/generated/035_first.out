Fatal error: Call to undefined method DtlIter::first() in /home/rune/workspace/php-misc/php-dtl-iterator/tests/generated/035_first.php on line 4